islandora.module

```diff
@@ -429,7 +429,9 @@ function islandora_admin_paths() {
   $paths['islandora/object/*/manage*'] = TRUE;
   $paths['islandora/object/*/delete*'] = TRUE;
   $paths['islandora/object/*/datastream/*/edit'] = TRUE;
-  $paths['islandora/object/*/datastream/*/versions'] = TRUE;
+  $paths['islandora/object/*/datastream/*/version*'] = TRUE;
```

Comment:
Question. Does this extra * at the end of version could open any issues of trying to resolve a menu item that does not exist? like versionABC? Just a question. I don't see any menu item resolving to anything else than 'versions'.  All the rest looks totally Ok for me.

Reply:
I can't really see it being a big issue; and this is the same behaviour as with the `manage` and `delete` paths just above anyway. I did it this way so we could handle the 4 version paths (`/version`, `/version/%/delete`, `/version/%/revert`, `/version/%/view`) all at once.

Reply:
@mghughes. In that case i would really prefer the verbose option, which means /version/%/delete, etc, i know it does not change things but it would make code easier to read.

Reply:
@DiegoPino No problem, that's done now.